core/interop/cats-mtl/src/main/scala/tofu/core/interop/catsmtl/implicits.scala

```diff
@@ -0,0 +1,67 @@
+package tofu.core.interop.catsmtl
+
+import cats.mtl.{Ask, Local, Handle => MHandle, Raise => MRaise}
+import cats.syntax.functor._
+import cats.{Applicative, Functor}
+import tofu.{Errors, Raise, WithContext, WithLocal}
+
+object implicits {
```

Comment:
Can we have separate `implicits` object for implicits and `instances` for explicit instances.
Names of the explicit ones should be _KPACUBbIE_.

Reply:
Should explicit methods have explicit arguments as well? (except Applicative ones)

Reply:
Good question. Let's try to think of a real example where it may be useful.

Reply:
I think we should place `TOFU`/`MTL` instance explicit and leave others as implicit.